lib/cypress/qrda_post_processor.rb

```diff
@@ -19,5 +21,29 @@ def self.select_negated_code(data_element, bundle)
              end
       data_element.dataElementCodes << code
     end
+
+    # create an issue message for any negations that are done with a single code rather than vs
+    def self.issues_for_negated_single_codes(patient, bundle, measures)
+      drc_codes = ValueSet.where(oid: /drc/i).map { |vs| vs.concepts.first.code }
+      error_list = []
+      patient.qdmPatient.dataElements.each do |de|
+        next unless de['negationRationale']
+
+        de.codes.each do |c|
+          next unless c.codeSystemOid != '1.2.3.4.5.6.7.8.9.10' && !drc_codes.include?(c)
```

Comment:
wooohooo. Measure Evaluator did its job!!!

!drc_codes.include?(**c**) should be !drc_codes.include?(**c.code**)